### ISSUE TITLE
Modeling the Distribution of Normal Data in Pre-Trained Deep Features for Anomaly Detection

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#122]
# TL;DR
A study of anomaly detection using a pretrained model. Fitting multivariate Gaussian distributions to multiple hidden layer features and detecting anomalies using the Mahalanobis' distance.They achieved SOTA with an MVTe dataset that is rather close to the real application environment.

![224 x 224](https://user-images.githubusercontent.com/24524018/93829888-ae2bcc80-fca9-11ea-8e6e-b9e78f8f1ba3.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.14140

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Oliver Rippel, Patrick Mertens, Dorit Merhof
- RWTH Aachen University

# Methods

# Results

# Comments
